apps/api/server.ts

```diff
@@ -0,0 +1,16 @@
+import { json } from 'body-parser';
+import cors from 'cors';
+
+import { verifyToken } from './middleware/verifyToken';
+
+export const configure = async (express) => {
+	const app = express();
+	app.use(json());
+	app.use(cors());
+
+	app.post('/api/signin', verifyToken, (req, res) => {
+		res.json(req.decodedToken);
+	});
```

Comment:
since we make `verifyToken` a nice-to-have, no-exception middleware, we can use it at the level of `json` or `cors` middleware, like so:
```typescript
app.use(json());
app.use(cors());
app.use(verifyToken()); // <-- in this context, rename the middleware e.g to `prepareAuthContext` would make more sense

app.post('/api/signin', (req, res) => {
		res.json(req.userProfile);
	});
```